src/katsdpcontroller/scheduler.py

```diff
@@ -3951,7 +3995,7 @@ def inverseOffers(self, driver, offers):
     def offerRescinded(self, driver, offer_id):
         if offer_id.value in self._resolving_offers:
             resolving_offer = self._resolving_offers.pop(offer_id.value)
-            resolving_offer.rescinded = True
+            resolving_offer.rescinded.add(offer_id.value)
             resolving_offer.task.cancel()
```

Comment:
@bmerry this shouldn't be here anymore - will cancel resolution on *all* lookups.